testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/organization/admin/OrganizationMemberTest.java

```diff
@@ -229,6 +231,48 @@ public void testGetAllDisabledOrganization() {
         }
     }
 
+    @Test
+    public void testGetAllDisabledOrganizationProvider() throws IOException {
+        OrganizationRepresentation orgRep = createOrganization();
+        OrganizationResource organization = testRealm().organizations().get(orgRep.getId());
+
+        // add some unmanaged members to the organization.
+        for (int i = 0; i < 5; i++) {
+            addMember(organization, "member-" + i + "@neworg.org");
+        }
+
+        // onboard a test user by authenticating using the organization's provider.
+        super.assertBrokerRegistration(organization, bc.getUserEmail());
+
+        // now fetch all users from the realm
+        List<UserRepresentation> members = testRealm().users().search("*neworg*", null, null);
+        members.stream().forEach(user -> assertThat(user.isEnabled(), is(Boolean.TRUE)));
+
+        // disable the organization provider
+        try (RealmAttributeUpdater rau = new RealmAttributeUpdater(testRealm())
+                .setOrganizationEnabled(Boolean.FALSE)
+                .update()) {
+
+            // now fetch all members from the realm - unmanaged users should still be enabled, but managed ones should not.
+            List<UserRepresentation> existing = testRealm().users().search("*neworg*", null, null);
+            assertThat(existing, hasSize(members.size()));
+            for (UserRepresentation user : existing) {
+                if (user.getEmail().equals(bc.getUserEmail())) {
+                    assertThat(user.isEnabled(), is(Boolean.FALSE));
+
+                    // try to update the disabled user (for example, try to re-enable the user) - should not be possible.
+                    user.setEnabled(Boolean.TRUE);
+                    try {
+                        testRealm().users().get(user.getId()).update(user);
```

Comment:
Is this the current behavior when you disable a regular user? I mean, AFAIK you can always updated a disabled user.

The key point is that they won't be able to authenticate.

Reply:
I believe it's not possible to update disabled user. This test uses the same approach as the `testGetAllDisabledOrganization()` which verifies that it's not possible to update the user in case the single organization is disabled.

Reply:
True, you are keeping the same behavior here. Forget about it then.

We can discuss this as a follow-up but I'm not 100% sure if we really want the current behavior ...

Reply:
We can discuss this, but currently any user disabled because the federated storage is disabled, or the organization is disabled, is read-only. A user that was individually disabled can still be updated.